lib/templater.js

```diff
@@ -31,6 +31,7 @@ function Templater(options) {
       dust.loadSource(compiled);
       deferred.resolve();
     } catch (error) {
+      console.error(error.message);
```

Comment:
Yes, I added it as currently if there are compile issues, there is no output to the console. This just makes debugging easier when someone breaks something in dust